### DISCUSSION TITLE
Plans to have Python3.8 Quickstart examples at https://docs.zitadel.ch/docs/quickstarts/introduction/ ?

### DISCUSSION THREAD

**Discussion by gc-ss** [zitadel/zitadel#2061] [Q&A]
Love this project!

Are there any plans to have Python3.8 Quickstart examples at https://docs.zitadel.ch/docs/quickstarts/introduction/?

**Comment by gc-ss** [comment]
If I were to start integrating with Zitadel in my Python3.8 project, what are some first steps I should take?

**Reply by fforootd** [reply]
Just for easy discovery we provide some examples here -> https://github.com/caos/zitadel-examples/tree/main/python3
